internal/cli/federatedauthentication/federationsettings/describe.go

```diff
@@ -64,9 +64,9 @@ func DescribeBuilder() *cobra.Command {
 	opts := new(DescribeOpts)
 	cmd := &cobra.Command{
 		Use:     "describe <orgID>",
-		Aliases: []string{"show"},
+		Aliases: []string{"get"},
```

Comment:
make sure to regenerate the doc

Reply:
did it but no changes `Code Health / docs (pull_request) Successful in 1m`

Reply:
(the test is hidden, I'll create the doc PR later with more commands)